packages/admin/admin/src/error/errordialog/createErrorDialogApolloLink.tsx

```diff
@@ -41,23 +45,24 @@ export const createErrorDialogApolloLink = () => {
             if (isServerError(networkError)) {
                 const { statusCode } = networkError;
                 httpStatus = `${statusCode} ${getReasonPhrase(statusCode)}`;
-
-                if (statusCode === StatusCodes.UNAUTHORIZED) {
-                    title = <FormattedMessage id="comet.errorDialog.sessionExpired.title" defaultMessage="Session expired" />;
-                    userMessage = (
-                        <>
-                            <Typography gutterBottom>
-                                <FormattedMessage id="comet.errorDialog.sessionExpired.message" defaultMessage="Your login-session has expired." />
-                            </Typography>
-                            <Button href="/" color="info" variant="outlined">
-                                <FormattedMessage id="comet.errorDialog.sessionExpired.button" defaultMessage="Re-login" />
-                            </Button>
-                        </>
-                    );
-                }
+                isUnauthenticated = statusCode === StatusCodes.UNAUTHORIZED;
             }
         }
 
+        if (!isUnauthenticated) {
```

Comment:
```suggestion
        if (isUnauthenticated) {
```

Shouldn't this show the error if a user is unauthenticated? Or do you ignore it because it's handled in the provider? Then the user message is incorrect